documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -91,7 +94,7 @@ kubectl logs -f _CLUSTER-OPERATOR-POD-NAME_ | grep -E "Kafka version upgrade fro
 +
```

Comment:
On line 80 ... I would probably say instead of _If the image for the Kafka version is different from the image defined in `STRIMZI_KAFKA_IMAGES` for the Cluster Operator, update `Kafka.spec.kafka.image`._ something like this:

_If the image for the Kafka cluster is defined in the Kafka custom resource in `Kafka.spec.kafka.image`, update it to point to a container image the with new Kafka version._

Reply:
Done. I replaced the wording for step 4.